packages/containers/containers.2.0+alpha1/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "1.2"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes"
+homepage: "https://github.com/c-cube/ocaml-containers/"
+bug-reports: "https://github.com/c-cube/ocaml-containers/issues/"
+doc: "https://c-cube.github.io/ocaml-containers"
+tags: ["stdlib" "containers" "iterators" "list" "heap" "queue"]
+dev-repo: "https://github.com/c-cube/ocaml-containers.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: [make "test"]
```

Comment:
Is it possible to use a jbuilder command to run the tests? There's no need to rely on make unless it's totally necessary for this.

Reply:
it's on master, but can we please not block the PR for this?